BUILD

```diff
@@ -1,37 +1,73 @@
 load("@rules_python//python:defs.bzl", "py_runtime_pair")
 load("@//tools:stamp.bzl", "stamp_build_setting")
-load("@pip_dependencies//:requirements.bzl", "entry_point")
+load("@aspect_bazel_lib//lib:expand_make_vars.bzl", "expand_template")
 
-# Needed by the custom toolchain
-# Can remove if we siwtch back to the canonical one
-"""
-py_runtime(
-    name = "python3_runtime",
-    files = ["@python_interpreter//:files"],
-    interpreter = "@python_interpreter//:python_bin",
-    python_version = "PY3",
+# <default py version change>: If the default version of python changes for Sematic, this
+# will need updating
+load("@pip_dependencies38//:requirements.bzl", entry_point38="entry_point")
+load("@python3_8//:defs.bzl", interpreter38="interpreter")
+
+
+exports_files(["README.rst"])
+
+stamp_build_setting(name = "stamp")
+
+alias(
+    name = "python_coverage_tools",
+    actual = entry_point38("coverage"),  # <default py version change>
     visibility = ["//visibility:public"],
 )
 
-py_runtime_pair(
-    name = "sematic_py_runtime_pair",
-    py2_runtime = None,
-    py3_runtime = ":python3_runtime",
+# The stuff going on here is fairly complicated. It's based on the
+# approach from https://blog.aspect.dev/many-python-versions-one-bazel-build
+# I've updated the stub.py.tpl to include a summarizing explanation.
+expand_template(
+    name = "interpreter_stub",
+    out = "stub.py",
+    data = [
+        # Reference to the binary used to run the stub.py script that
+        # launches the "real" interpreter
+        interpreter38,  # <default py version change>
+    ],
+    is_executable = True,
+    substitutions = {
+        # Template the path to the default interpreter.
+        # Can't be hardcoded in stub.py because we want it to point at
+        # the hermetic default interpreter, and the hermetic default
+        # interpreter's path is determined by bazel.
+        "%DEFAULT_PYTHON_INTERPRETER_PATH%": "$(execpath {})".format(interpreter38),
+    },
+    template = "//:stub.py.tpl",
+    visibility = ["//visibility:public"],
 )
 
 toolchain(
-    name = "sematic_py_toolchain",
-    toolchain = ":sematic_py_runtime_pair",
+    name = "py_stub_toolchain",
+    toolchain = ":py_stub_runtime_pair",
     toolchain_type = "@bazel_tools//tools/python:toolchain_type",
 )
-"""
-
-exports_files(["README.rst"])
```

Comment:
Why removing this? This is needed by the wheel target. Unless it isn't?

Reply:
I didn't remove it, I just moved it